Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -53,6 +55,20 @@
 
 #include "UICommon/GameFile.h"
 
+#if defined(_WIN32)
+static bool FFmpegWin32Exists()
+{
+  const std::string avcodec = File::GetExeDirectory() + "\\" + "avcodec-58.dll";
```

Comment:
This can just use `DIR_SEP` or `DIR_SEP_CHR` from CommonPaths, it doesn't need to be a backslash.